### DISCUSSION TITLE
Sidebar not getting created!

### DISCUSSION THREAD

**Discussion by cbsecompass** [thuliteio/doks#1247] [How-to guides] [answered]
I'm a complete beginner here. I'm using the Doks template for my site.

I'm trying to run a second Docs-style page for my site, but for some reason, the sidebar doesn't appear. These are the commands I've run so far.

`hugo new --kind docs class10
`
This created a lorem/ipsum file within class10.

`hugo new ./content/class10/introduction/getting-started.md
`
Then I edited the sectionNav at params.toml

`sectionNav = ["docs", "Class10"]
`
Then in menu.en.toml I added this

```
[[main]]

name = "Class 10"

url = "/class10/introduction/getting-started/"

weight = 20
```

The sidebar still doesn't appear.

Images attached of the resulting page. View them here. [https://imgur.com/a/RW3MSEg](url)

**Comment by h-enk** [comment]
I can't replicate it.  I followed your instructions and it works for me (http://localhost:1313/class10/introduction/getting-started/):

![2024-06-03_08-45-08](https://github.com/gethyas/doks/assets/3902872/4c37ac2d-bf9b-4f4b-9009-05b09e490c84)

- Did you set `draft: false` in the frontmatter (I guess you did)?
- Did you clear the Hugo build directories? Do something like this:

```bash
npx shx rm -rf public resources .hugo_build.lock && npm run dev
```

- Please share your repo, so I can have a better look at it

**Comment by cbsecompass** [comment]
yes. I have set draft: false

https://github.com/cbsecompass/querycompass.git

I wasn't able to add the resources folder as there were too many files. Hope this is alright? Please let me know if it isn't

**Reply by h-enk** [reply, marked as answer]
In `config/_default/params.toml` you've set `sectionNav = ["docs", "Class10"]` — see https://github.com/cbsecompass/querycompass/blob/main/config/_default/params.toml#L47

Make the section name all lowercase and you're good:

```toml
  # Nav
  sectionNav = ["docs", "class10"] # ["docs"] (default) or list of sections (e.g. ["docs", "guides"])
```

> I wasn't able to add the resources folder as there were too many files. Hope this is alright? Please let me know if it isn't

That's okay. Only handy when you've got lots of images — for more background, see [Image processing performance consideration](https://gohugo.io/content-management/image-processing/#image-processing-performance-consideration)

**Reply by cbsecompass** [reply]
OMG, that's genius! Thank you SO much. Been racking my brain for something so elementary.
